src/components/MovieDetail/index.ts

```diff
@@ -0,0 +1,116 @@
+import { MovieDetailResponse } from "../../types";
+import starImgFilled from "../../assets/star_filled.png";
+import starImgEmpty from "../../assets/star_empty.png";
+import { $, $$ } from "../../utils/selector";
+
+class MovieDetail {
+  constructor() {}
+
+  static render(movieDetail: MovieDetailResponse) {
+    const { id, title, genres, overview, poster_path, vote_average } =
+      movieDetail;
```

Comment:
```suggestion
  static render({
    id,
    title,
    genres,
    overview,
    poster_path,
    vote_average,
  }: MovieDetailResponse) {
```

바로 destructuring 하시죠~